queue_job/models/queue_job.py

```diff
@@ -418,6 +418,61 @@ def requeue_stuck_jobs(self, enqueued_delta=5, started_delta=0):
         ).requeue()
         return True
 
+    def fail_dead_jobs(self, started_delta, force_low_delta=False):
```

Comment:
Not a required change, but I think it would be reasonable to `force_low_delta=True` here to turn your safety check on by default. I'm going to override
```xml
            <field name="code">model.fail_dead_jobs(240)</field>
```
anyway to significantly decrease the started_delta for my own purposes, so it will be no inconvenience to me to also disable that check.

Reply:
It's the opposite, you will have to change the cron to something like : 
```py
model.fail_dead_jobs(5, force_low_delta=True)
```

BTW, why are you planning to put a low value here ? What's your use case ?